index.js

```diff
@@ -1,3 +1,4 @@
+const API_KEY = 'iBJukOptIxcWGqoclgSIDylXcBS7qpIUncziO0GW'
```

Comment:
Can you put the API_KEY into the key.js, and ignore key.js so other people won't see it?

Reply:
yes, thank you! Sorry I keep forgetting to remove it but hearing this workaround is super helpful